domain/src/main/java/org/fao/geonet/domain/HarvestHistory.java

```diff
@@ -311,29 +297,30 @@ public Element getInfoAsXml() throws IOException, JDOMException {
     private Element checkInfoXml(Element infoAsXml) {
         List<Element> logfileElements = infoAsXml.getChildren("logfile");
         // check logfiles present to ensure path exists
-        if (logfileElements.size() > 0) {
-            if (logfileElements.size() != 1) {
-                Log.debug(Constants.DOMAIN_LOG_MODULE, "Harvest history unexpectedly lists multiple logfiles: " + logfileElements.size());
-            }
-            boolean isLogFileFound = false;
-            for (Iterator iter = logfileElements.iterator(); iter.hasNext();) {
-                Element logfile = (Element) iter.next();
-                String path = logfile.getText();
-                File file = new File(path);
-                if (file.exists() && file.canRead()) {
-                    if (isLogFileFound) {
-                        // we already have one logfile
-                        Log.debug(Constants.DOMAIN_LOG_MODULE, "Ignoring add unexpected logfile: `" + path + "`");
-                        iter.remove();
-                    } else {
-                        isLogFileFound = true;
-                    }
-                } else {
-                    Log.debug(Constants.DOMAIN_LOG_MODULE, "Harvest history logfile `" + path + "` ignored, no longer available");
-                    iter.remove();
-                }
-            }
-        }
+        // TODO: Would require to check in the app log folder
+//        if (logfileElements.size() > 0) {
+//            if (logfileElements.size() != 1) {
+//                Log.debug(Constants.DOMAIN_LOG_MODULE, "Harvest history unexpectedly lists multiple logfiles: " + logfileElements.size());
+//            }
+//            boolean isLogFileFound = false;
+//            for (Iterator iter = logfileElements.iterator(); iter.hasNext();) {
+//                Element logfile = (Element) iter.next();
+//                String path = logfile.getText();
+//                File file = new File(path);
+//                if (file.exists() && file.canRead()) {
+//                    if (isLogFileFound) {
+//                        // we already have one logfile
+//                        Log.debug(Constants.DOMAIN_LOG_MODULE, "Ignoring add unexpected logfile: `" + path + "`");
+//                        iter.remove();
+//                    } else {
+//                        isLogFileFound = true;
+//                    }
+//                } else {
+//                    Log.debug(Constants.DOMAIN_LOG_MODULE, "Harvest history logfile `" + path + "` ignored, no longer available");
+//                    iter.remove();
+//                }
+//            }
+//        }
```

Comment:
Can this cause any issue or it's just to store the log file content in the database?

Reply:
A lot of this code was to recover from the times when multiple logfiles were listed, not sure why that is being commented out.